kubernetes-client-api/src/main/java/io/fabric8/kubernetes/client/utils/KubernetesResourceUtil.java

```diff
@@ -440,13 +473,6 @@ public static Secret createDockerRegistrySecret(String dockerServer, String user
     credentials.put("auth", Base64.getEncoder().encodeToString(usernameAndPasswordAuth.getBytes(StandardCharsets.UTF_8)));
     auths.put(dockerServer, credentials);
     dockerConfigMap.put("auths", auths);
-
-    String dockerConfigAsStr = Serialization.jsonMapper().writeValueAsString(dockerConfigMap);
-
-    return new SecretBuilder()
-        .withNewMetadata().withName("harbor-secret").endMetadata()
-        .withType("kubernetes.io/dockerconfigjson")
-        .addToData(".dockerconfigjson", Base64.getEncoder().encodeToString(dockerConfigAsStr.getBytes(StandardCharsets.UTF_8)))
-        .build();
+    return dockerConfigMap;
```

Comment:
I am not sure to understand how this is replaced with a new code with a default name (or custom)